mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -2,6 +2,11 @@ CLASS net/minecraft/class_3620 net/minecraft/block/MapColor
 	COMMENT Represents the surface color of a block when rendered from the {@link net.minecraft.client.render.MapRenderer}.
 	COMMENT Color names refer to a material or an object which refers to their vanilla Minecraft textures, not their real-world counterparts, eg. "emerald green".
 	COMMENT Names are in the form of either <i>blockReference_baseColor</i> or <i>color</i>.
+	COMMENT
+	COMMENT <p>When the map is rendered, the {@code MapColor.Brightness#brightness} value is added to the
```

Comment:
```suggestion
	COMMENT <p>When the map is rendered, the {@link MapColor.Brightness#brightness} value is added to the
```
Use `@link` to actually refer to fields, methods, or classes (also packages). The `@code` effect is just like `this` (github editor double `` backtick)